performance_regression_operator_multi_tenant_test.py

```diff
@@ -47,7 +47,7 @@ def get_str_index(self):
         return f"k8s-perf-{self.db_cluster.k8s_cluster.tenants_number}-tenants"
 
     def id(self):  # pylint: disable=invalid-name
-        return self.test_config.test_id() + f"-{self._test_index}"
+        return self._test_index
```

Comment:
i'm not sure it's enough, since now still doesn't have the full qualified name of the test in id
meaning that if you'll have multiple type of tests the results might overlap.

let have some runs, and then start working on the analysis and report, you might want to circle back to this.